v1/models/gpu/index.js

```diff
@@ -0,0 +1,226 @@
+            "depthBiasClamp": Boolean,
+            "fillModeNonSolid": Boolean,
+            "depthBounds": Boolean,
+            "wideLines": Boolean,
+            "largePoints": Boolean,
+            "alphaToOne": Boolean,
+            "multiViewport": Boolean,
+            "samplerAnisotropy": Boolean,
+            "textureCompressionETC2": Boolean,
+            "textureCompressionASTC_LDR": Boolean,
+            "textureCompressionBC": Boolean,
+            "occlusionQueryPrecise": Boolean,
+            "pipelineStatisticsQuery": Boolean,
+            "vertexPipelineStoresAndAtomics": Boolean,
+            "fragmentStoresAndAtomics": Boolean,
+            "shaderTessellationAndGeometryPointSize": Boolean,
+            "shaderImageGatherExtended": Boolean,
+            "shaderStorageImageExtendedFormats": Boolean,
+            "shaderStorageImageMultisample": Boolean,
+            "shaderStorageImageReadWithoutFormat": Boolean,
+            "shaderStorageImageWriteWithoutFormat": Boolean,
+            "shaderUniformBufferArrayDynamicIndexing": Boolean,
+            "shaderSampledImageArrayDynamicIndexing": Boolean,
+            "shaderStorageBufferArrayDynamicIndexing": Boolean,
+            "shaderStorageImageArrayDynamicIndexing": Boolean,
+            "shaderClipDistance": Boolean,
+            "shaderCullDistance": Boolean,
+            "shaderFloat64": Boolean,
+            "shaderInt64": Boolean,
+            "shaderInt16": Boolean,
+            "shaderResourceResidency": Boolean,
+            "shaderResourceMinLod": Boolean,
+            "sparseBinding": Boolean,
+            "sparseResidencyBuffer": Boolean,
+            "sparseResidencyImage2D": Boolean,
+            "sparseResidencyImage3D": Boolean,
+            "sparseResidency2Samples": Boolean,
+            "sparseResidency4Samples": Boolean,
+            "sparseResidency8Samples": Boolean,
+            "sparseResidency16Samples": Boolean,
+            "sparseResidencyAliased": Boolean,
+            "variableMultisampleRate": Boolean,
+            "inheritedQueries": Boolean
+        },
+        "memory": {
+            "memoryTypeCount": Number,
+            "memoryTypes": [{
+                "heapIndex": Number,
+                "propertyFlags": [String]
+            }],
+            "memoryHeapCount": Number,
+            "memoryHeaps": [{
+                "heapIndex": Number,
+                "propertyFlags": [String]
+            }]
+        },
+        "queues": [{
+            "minImageTransferGranularity": {
+                "depth": Number,
+                "height": Number,
+                "width": Number
+            },
+            "queueCount": Number,
+            "queueFlags": [String],
+            "timestampValidBits": Number
+        }],
+        "extensions": [{
+            "extensionName": String,
+            "specVersion": Number
+        }],
+        "formats": [mongoose.Schema.Types.Formats]
+    });
+    return mongoose.model('gpu', schema);
```

Comment:
`gpu` => `Gpu`, as schema should begin with an uppercase letter, do not change the folder though